packages/cli/lib/lib/webpack/create-load-manifest.js

```diff
@@ -1,15 +1,22 @@
-module.exports = assets => {
+const isESM = filename => /\.esm\.js$/.test(filename);
```

Comment:
Not entirely scoped to this PR, but would you be open to using `.mjs`?

Reply:
I am open to do this in another PR. IIRC webpack's uglify only runs on .just files by default so I switched.